integration-cli/docker_cli_run_unix_test.go

```diff
@@ -242,13 +242,6 @@ func (s *DockerSuite) TestRunEchoStdoutWithCPUSharesAndMemoryLimit(c *check.C) {
 	}
 }
 
-func (s *DockerSuite) TestRunWithCpuset(c *check.C) {
```

Comment:
What was this test removed? isn't `--cpuset` still valid?  Its the single-dash option that was removed.